WatchFaceAlphaKotlin/app/src/main/java/com/example/android/wearable/alpha/data/watchface/ColorStyleIdAndResourceIds.kt

```diff
@@ -0,0 +1,129 @@
+ */
+sealed class ColorStyleIdAndResourceIds(
+    val id: String,
+    val name: Int,
+    val iconResourceId: Int,
+    val complicationStyleDrawableId: Int,
+    val primaryColorId: Int,
+    val secondaryColorId: Int,
+    val backgroundColorId: Int,
+    val outerElementColorId: Int
+) {
+    object Ambient : ColorStyleIdAndResourceIds(
+        id = AMBIENT_COLOR_STYLE_ID,
+        name = AMBIENT_COLOR_STYLE_NAME_RESOURCE_ID,
+        iconResourceId = AMBIENT_COLOR_STYLE_ICON_ID,
+        complicationStyleDrawableId = R.drawable.complication_white_style,
+        primaryColorId = R.color.ambient_primary_color,
+        secondaryColorId = R.color.ambient_secondary_color,
+        backgroundColorId = R.color.ambient_background_color,
+        outerElementColorId = R.color.ambient_outer_element_color
+    )
+
+    object Red : ColorStyleIdAndResourceIds(
+        id = RED_COLOR_STYLE_ID,
+        name = RED_COLOR_STYLE_NAME_RESOURCE_ID,
+        iconResourceId = RED_COLOR_STYLE_ICON_ID,
+        complicationStyleDrawableId = R.drawable.complication_red_style,
+        primaryColorId = R.color.red_primary_color,
+        secondaryColorId = R.color.red_secondary_color,
+        backgroundColorId = R.color.red_background_color,
+        outerElementColorId = R.color.red_outer_element_color
+    )
+
+    object Green : ColorStyleIdAndResourceIds(
+        id = GREEN_COLOR_STYLE_ID,
+        name = GREEN_COLOR_STYLE_NAME_RESOURCE_ID,
+        iconResourceId = GREEN_COLOR_STYLE_ICON_ID,
+        complicationStyleDrawableId = R.drawable.complication_green_style,
+        primaryColorId = R.color.green_primary_color,
+        secondaryColorId = R.color.green_secondary_color,
+        backgroundColorId = R.color.green_background_color,
+        outerElementColorId = R.color.green_outer_element_color
+    )
+
+    object Blue : ColorStyleIdAndResourceIds(
+        id = BLUE_COLOR_STYLE_ID,
+        name = BLUE_COLOR_STYLE_NAME_RESOURCE_ID,
+        iconResourceId = BLUE_COLOR_STYLE_ICON_ID,
+        complicationStyleDrawableId = R.drawable.complication_blue_style,
+        primaryColorId = R.color.blue_primary_color,
+        secondaryColorId = R.color.blue_secondary_color,
+        backgroundColorId = R.color.blue_background_color,
+        outerElementColorId = R.color.blue_outer_element_color
+    )
+
+    object White : ColorStyleIdAndResourceIds(
+        id = WHITE_COLOR_STYLE_ID,
+        name = WHITE_COLOR_STYLE_NAME_RESOURCE_ID,
+        iconResourceId = WHITE_COLOR_STYLE_ICON_ID,
+        complicationStyleDrawableId = R.drawable.complication_white_style,
+        primaryColorId = R.color.white_primary_color,
+        secondaryColorId = R.color.white_secondary_color,
+        backgroundColorId = R.color.white_background_color,
+        outerElementColorId = R.color.white_outer_element_color
+    )
+
+    companion object {
+        // Translates the string id to the correct ColorStyleIdAndResourceIds object.
```

Comment:
Nit: Here and a few other places, the `//` comment won't show up in the documentation for the function: 
```suggestion
        /**
         * Translates the string id to the correct ColorStyleIdAndResourceIds object.
         */
```

Reply:
Addressed